Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/GrabberCameraPipeline.java

```diff
@@ -64,12 +64,14 @@ public Mat processFrame(Mat input) {
             xPosition = boundingRect.x + (boundingRect.width * 0.5);
             yPosition = boundingRect.y + (boundingRect.height * 0.5);
 
+            //TODO: add a cutoff point only for the bottom quarter of the frame as to not see the wheels
+
             // calculate distance of the center of the box from the center of the camera
             double distanceFromCenter = Math.sqrt(Math.pow(Math.abs((xPosition - Constants.CAMERA_CENTER_X)), 2) + Math.pow(Math.abs((yPosition - Constants.CAMERA_CENTER_Y)), 2));
 
             // determine if the detected area is close enough to the center
             // this is done to avoid detecting the wheels which are also black
-            if (distanceFromCenter < Constants.DISTANCE_FROM_CENTER_JUNCTION_TOP) {
+            if (xPosition >= 300) {
```

Comment:
Where is 300 coming from? Use a constant

Reply:
Replaced with a constant